python/CLOUDFRONT_VIEWER_POLICY_HTTPS/CLOUDFRONT_VIEWER_POLICY_HTTPS_test.py

```diff
@@ -0,0 +1,206 @@
+import json
+import sys
+import unittest
+try:
+    from unittest.mock import MagicMock
+except ImportError:
+    from mock import MagicMock
+import botocore
+
+##############
+# Parameters #
+##############
+
+# Define the default resource to report to Config Rules
+DEFAULT_RESOURCE_TYPE = 'AWS::CloudFront::Distribution'
+
+#############
+# Main Code #
+#############
+
+CONFIG_CLIENT_MOCK = MagicMock()
+STS_CLIENT_MOCK = MagicMock()
+
+class Boto3Mock():
+    @staticmethod
+    def client(client_name, *args, **kwargs):
+        if client_name == 'config':
+            return CONFIG_CLIENT_MOCK
+        if client_name == 'sts':
+            return STS_CLIENT_MOCK
+        raise Exception("Attempting to create an unknown client")
+
+sys.modules['boto3'] = Boto3Mock()
+
+RULE = __import__('CLOUDFRONT_VIEWER_POLICY_HTTPS')
+
+class SampleTest(unittest.TestCase):
```

Comment:
One last change from my end, change the name to ComplianceTest

Reply:
Done !!